.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+---
+name: Bug report
+about: Let us know about an unexpected error, a crash, or an incorrect behavior.
+
+---
+
```

Comment:
Maybe add a section like:
*Installation Process*
Did you install polaris on your local machine, or in a kubernetes cluster? Did you use kubectl, helm, brew, a GitHub release, or the source code?

Reply:
Good suggestion. I added the installation process section in the bug report template.